Documentation/Development.md

```diff
@@ -134,81 +134,6 @@ Where "N" configures the log verbosity and is one of the following numbers: 0 (e
 
 As much as is practical, all code should be covered by tests. New tests can be added under the `Tests` directory and should use `XCTest`. The rest of this section will describe the additional tools available in the `SKTestSupport` module to make it easier to write good and efficient tests.
 
```

Comment:
I assume your opinion here is that the tests are now obvious enough without any description?

Reply:
My opinion here is that we should write more documentation that’s up-to-date.

This paragraph in particular doesn’t add any value IMO. It’s mostly just: There should be tests and we use XCTest, which, I think is obvious.

Reply:
OK, for those, I would say that 

> As much as is practical, all code should be covered by tests. New tests can be added under the `Tests` directory and should use `XCTest`.

is obvious and not something that needs to be mentioned in documentation.